web/styles/settings.css

```diff
@@ -399,6 +400,40 @@ td .button {
     }
 }
 
+/* Class for displaying an input with an
+   auxiliary control, e.g., the play button
+   next to the notification sound dropdown. */
+.input-element-with-control {
+    display: flex;
+    align-items: center;
+    /* Preserve the effective space allotted
+       inside the input group.
+
+       Select element: 325px
+       Button element: ~29px
+       Right margin:     4px
+       TOTAL           358px */
+    max-width: 358px;
```

Comment:
Not critical for this PR but I think we should try to get into the habit of extracting CSS variables for magic values like this 325px using `calc` to combine them as appropriate.

Reply:
Agreed, although in this specific case, I'd start by rewriting the styles around the select widths, which are currently, misleadingly set via `min-width`. Ideally, no one should have to care here about this 358px width; you'd just let flexbox do its thing and let the layout take care of itself.